### PR TITLE
[PBW-2961] responsive control bar

### PR DESCRIPTION
**Pull request by ksunika**

### CODE REVIEW

**Comment by mistermendez** [comment]
@ksunika conflicts

**Comment by ksunika** [comment]
@mistermendez conflicts resolved

File: js/components/controlBar.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ var React = require('react'),
 var ControlBar = React.createClass({
   getInitialState: function() {
     this.isMobile = this.props.controller.state.isMobile;
+    this.responsiveUIMultiple = Utils.responsiveUIMultiple(this.props.responsiveView);
     return {
```
Comment (mistermendez) [inline js/components/controlBar.js]: could you get multiple from skin.json?

`this.props.skinConfig.responsive.breakpoints.sm.multiple`

File: js/skin.js [hunk 2]
```diff
@@ -34,7 +34,8 @@ var Skin = React.createClass({
   componentDidUpdate: function() {
     // Notify AMC the correct overlay rendering info
     if (this.state.screenToShow !== null && !this.overlayRenderingEventSent) {
-      var marginHeight = Utils.getScaledControlBarHeight(this.getDOMNode().clientWidth) + CONSTANTS.UI.defaultScrubberBarHeight;
+      var responsiveClass = ClassNames(this.generateBreakpointClasses());
+      var marginHeight = Utils.getScaledControlBarHeight(this.getDOMNode().clientWidth, responsiveClass) + Utils.responsiveUIMultiple(responsiveClass) * CONSTANTS.UI.defaultScrubberBarHeight;
       this.props.controller.publishOverlayRenderingEvent(marginHeight);
```
Comment (mistermendez) [inline js/skin.js]: this.getDOMNode().clientWidth not being used in Util.getScaledControlBarHeight()

File: js/components/controlBar.js (hunk 1 above)
Comment (mistermendez) [inline js/components/controlBar.js]: the multiple is available in the skin.json -

`this.props.skinConfig.responsive.breakpoints.sm.multiple`

**Comment by mistermendez** [comment]
Can we make this responsive w/o using CONSTANTS.UI.defaultControlBarHeight and use Sass/CSS.

```
.controlBar {
    height: $control-bar-height;
}

.small {
    .controlBar {
        $control-bar-height * $small-multiple;
}
}
```

**Comment by mistermendez** [comment]
Same with `CONSTANTS.UI.defaultScrubberBarPaddingHeight`. There seems to be a lot of unnecessary calculations and Util functions.

**Comment by ksunika** [comment]
@mistermendez, I removed unnecessary calculations in Utils, got rid of some of CONSTANTS.UI (3 are still left since they participate in calculations).

**Comment by mistermendez** [comment]
Sweet :candy: 

Looks good, except 3 unit test are broken (controlBar, moreOptions, scrubberBar)

**Comment by mistermendez** [comment]
:+1:

**Pull request merged**